src/createThemeFile.ts

```diff
@@ -19,8 +18,8 @@ function createThemeFile({
 	renderFiles: any;
 }): void {
 	const mainThemeFileName = `theme.ts`;
-	const relativeThemeFilePath = join(themesDirectory, mainThemeFileName);
-	const absoluteThemeFilePath = join(process.cwd(), relativeThemeFilePath);
+	const relativeThemeFilePath = [themesDirectory, mainThemeFileName].join('/');
```

Comment:
nit: template literals are probably nicer here `${themesDirectory}/${mainThemeFileName}`

Reply:
Yeah, I didn't end up needing to touch this line, so I reverted it. Had missed some issues with the unit tests that led me to take a different path.